packages/transaction-preview/src/index.ts

```diff
@@ -269,13 +269,48 @@ const mountTransactionPreview = (simResponse: MultiSimOutput) => {
 
   transactionPreviewDomElement.style.all = 'initial'
 
+  target.innerHTML = `
+    <style>
+      :host {
+        /* FONTS */
+        --font-family-normal: Sofia Pro;
+
+        --font-size-4: 1.25rem;
+        --font-size-5: 1rem;
+        --font-size-6: .875rem;
+        --font-size-7: .75rem;
+
+        --font-line-height-4: 12px;
+
+        --success-500: #5aec99;
+        --danger-400: #ff8080;
+
+        /* SPACING */
+        --spacing-4: 1rem;
+
+        /* BORDER RADIUS */
+        --border-radius-5: 8px;
+
+        /* SHADOWS */
+        --shadow-1: 0px 4px 12px rgba(0, 0, 0, 0.1);
+
+        /* MODAL POSITIONING */
+        --tp-z-index: 10;
+
+      }
+    </style>
+  `
```

Comment:
Any chance we can move this css to the style tag in packages/transaction-preview/src/views/Index.svelte ?

Reply:
Good call, done!